frontend/packages/vkt/public/i18n/koodisto/langs/koodisto_langs_en-GB.json

```diff
@@ -0,0 +1,204 @@
+{
+  "vkt": {
```

Comment:
koodiston kieliä ei varmaan tarvitse VKT:hen tuoda?

Edit:
Aivan, näitä käytetään tosiaan tutkintokielten kääntämisiin. Ne tosin tarkoituksella `common.examLanguage` alla. Kieliä ei kuitenkaan tule olemaan muita kuin suomi ja ruotsi (ja ehkä tulevaisuudessa joitain muitakin).

Reply:
Aa okei, poistan ton koodiston sitten ja nappaan tuolta common.examLanguagesta